src/canvas/braillecanvas.jl

```diff
@@ -93,8 +93,9 @@ function pixel!(c::BrailleCanvas, pixel_x::Int, pixel_y::Int, color::UserColorTy
     0 <= pixel_x <= c.pixel_width || return c
     0 <= pixel_y <= c.pixel_height || return c
     char_x, char_y, char_x_off, char_y_off = pixel_to_char_point(c, pixel_x, pixel_y)
-    c.grid[char_x, char_y] =
-        Char(UInt64(c.grid[char_x, char_y]) | UInt64(braille_signs[char_x_off, char_y_off]))
+    if 0x2800 <= (val = UInt64(c.grid[char_x, char_y])) <= 0x28ff
+        c.grid[char_x, char_y] = Char(val | UInt64(braille_signs[char_x_off, char_y_off]))
+    end
```

Comment:
Both versions are encrypted to be... 😢

Reply:
Sorry, I didn't get that, what versions ?

Reply:
The master version and the PR version... I'm afraid that I can't review this PR.

Reply:
Oh you mean you do not understand the changes ? Let me explain: `0x2800` is [blank braille](https://unicode-table.com/en/2800/), `0x28ff` is [full braille](https://unicode-table.com/en/28FF/). Doing a `binary or` is the low level operation for 'filling' the braille pattern with dots. So I'm just checking here that the value is a braille unicode char, before applying the increment.

Reply:
I've clarified this by using `const`s.